code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -53,13 +53,22 @@
 	fire_sound = 'sound/magic/staff_door.ogg'
 
 /obj/item/gun/magic/staff/slipping
-	name = "staff of slipping"
+	name = "wand of slipping"
 	desc = "An artefact that spits... bananas?"
 	ammo_type = /obj/item/ammo_casing/magic/slipping
 	icon_state = "staffofslipping"
 	item_state = "staffofslipping"
 	fire_sound = 'sound/items/bikehorn.ogg'
 
+/obj/item/gun/magic/staff/slipping/ruin
+	name = "staff of slipping"
+	desc = "An artefact that spits... bananas? This one appears to be low on charge..."
+	ammo_type = /obj/item/ammo_casing/magic/slipping
+	icon_state = "staffofslipping"
+	item_state = "staffofslipping"
```

Comment:
This is an override of `/obj/item/gun/magic/staff/slipping`, so you dont need there
```suggestion

```

Reply:
Done. I removed the old staff code and made a wand instead